src/widgets/medications/medications-summary.component.tsx

```diff
@@ -34,12 +34,14 @@ export default function MedicationsSummary(props: MedicationsOverviewProps) {
       name={t("Active Medications", "Active Medications")}
       match={props.match}
       styles={{ width: "100%", maxWidth: "45rem" }}
-      link={`/patient/${patientUuid}/chart/medications`}
+      link={`/patient/${patientUuid}/chart/medicationsleveltwo`}
```

Comment:
```suggestion
      link={`/patient/${patientUuid}/chart/medications`}
```

Reply:
We already have an endpoint with the  name medication

Reply:
see @jonathandick comment above

Reply:
@fatmali Please review, the changes requested by @jonathandick  have been effected.